src/components/pages/coach/GenerateReport/index.jsx

```diff
@@ -168,26 +173,38 @@ class GenerateReport extends React.Component{
     fetchUserData = async () => {
         await this.props.client.query({
             query: GET_USERDATA,
-            variables: { "token": localStorage.getItem("fprint"), "id": this.state.user.id }
+            variables: { "token": localStorage.getItem("fprint"), "id": this.state.uid }
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋